WalletWasabi.Tests/UnitTests/WabiSabi/Models/MultipartyTransactionTests.cs

```diff
@@ -0,0 +1,327 @@
+using NBitcoin;
+using System;
+using System.Collections.Immutable;
+using System.Linq;
+using WalletWasabi.Helpers;
+using WalletWasabi.WabiSabi.Backend.Models;
+using WalletWasabi.WabiSabi.Models;
+using WalletWasabi.WabiSabi.Models.MultipartyTransaction;
+using WalletWasabi.Tests.Helpers;
+using Xunit;
+
+namespace WalletWasabi.Tests.UnitTests.WabiSabi.Models
+{
+	public class MultipartyTransactionTests
+	{
+		private static MoneyRange DefaultAllowedAmounts = new (Money.Zero, Money.Coins(1));
+		private static Parameters DefaultParameters = new (FeeRate.Zero, DefaultAllowedAmounts, DefaultAllowedAmounts, Network.Main);
+
+		private static void ThrowsProtocolException(WabiSabiProtocolErrorCode expectedError, Action action) =>
+			Assert.Equal(expectedError, Assert.Throws<WabiSabiProtocolException>(action).ErrorCode);
+
+		[Fact]
+		public void TwoPartyNoFees()
+		{
+			using Key key1 = new();
+			using Key key2 = new();
+
+			var alice1 = WabiSabiFactory.CreateAlice(key: key1);
+			var alice2 = WabiSabiFactory.CreateAlice(key: key2);
+
+			var alice1Coin = alice1.Coins.First();
+			var alice2Coin = alice2.Coins.First();
+
+			var state = new Construction(DefaultParameters);
+
+			Assert.Empty(state.Inputs);
+			Assert.Empty(state.Outputs);
+
+			var oneInput = state.AddInput(alice1Coin);
```

Comment:
```suggestion
			var oneInputState = state.AddInput(alice1Coin);
```
?

Reply:
i find the readability of the shorter version a little bit better, there is nothing apart from manipulation of transaction states in the tests so it feels redundant to qualify, and i would prefer to qualify via the type as `State oneInput` instead if it really needs to be disambiguated

Reply:
This is just a suggestion, you don't need to modify it at all. However, `State oneInput` sounds good to me too.